changestopiglatin.txt

```diff
@@ -0,0 +1,13 @@
+
+pyg = 'ay'
+
+original = raw_input('Enter a word:')
+
+if len(original) > 0 and original.isalpha():
+    word = original.lower()
+    first = word[0]
+    new_word = word+first+pyg
+    new_word = new_word[1:len(new_word)]
```

Comment:
You can skip the `len(new_word)` when you want to show the entire rest by just typing `new_word[1:]`